perun-core/src/main/java/cz/metacentrum/perun/core/api/UsersManager.java

```diff
@@ -795,6 +797,57 @@ UserExtSource getUserExtSourceByExtLogin(PerunSession perunSession, ExtSource so
 	 */
 	boolean isLoginAvailable(PerunSession sess, String loginNamespace, String login) throws InvalidLoginException;
 
+	/**
+	 * Returns all pairs of blocked login in namespace (if namespace is null, then this login is blocked globally)
+	 *
+	 * @param sess
+	 */
```

Comment:
I know it's a technicality, but we could add @return tag into javadoc as well. 

Also, closer specification of returned object in the method description might be useful for future use, for this one let's say something like: List<Pair< LOGIN, NAMESPACE >>